README.md

```diff
@@ -1,5 +1,38 @@
 [![Build Status](https://travis-ci.org/freifunk/directory.api.freifunk.net.svg?branch=master)](https://travis-ci.org/freifunk/directory.api.freifunk.net)
 
+Freifunk Community Directory
+==========
+
+Dieses Github Repository enthält das [Community Directory](https://freifunk.net/blog/2013/12/die-freifunk-api/) der [Freifunk Initiative](https://freifunk.net/).
+Das Community Directory wird von zahlreichen Anwendungen / Apps verwendet, z.B. von der [Freifunk Karte](https://www.freifunk-karte.de/), der [Community Website](http://community.freifunk.net) oder der [App Freifunk](https://apps.apple.com/de/app/freifunk/id687078875)) für das iPhone oder das iPad.
+
+Dein Source Code Beitrag
+==========
+
+Erstelle einfach einen Pull Request, wenn Du Deine Community hier eintragen oder einen bestehenden Eintrag ändern bzw. löschen willst.
+
+Voraussetzungen:
+----------------
+1. Wir können nur Einträge annehmen, bei denen sichergestellt ist, dass diese langfristig gepflegt werden. Deine Community hat mindestens. 2-3 aktive Personen die diesen Ort betreuen (sie müssen keine Anwohner dieses Ortes sein).
+2. Deine Community hat mehrere Router in diesem Ort online.
+3. Dein Eintrag im Community Directory entspricht den [Qualitätskriterien](QUALITY.md)
+4. Deine Community API Datei entspricht den [Qualitätskriterien](QUALITY.md)
+5. Organisatorisch stellt Deine Community sicher, dass die Ansprechpartner (Team) binnen 7 Tagen auf Anfragen antworten und binnen 28 Tagen Probleme beheben können. 
+
+Zu Punkt 5 empfehlen wir:
+- E-Mail-Verteiler statt persönliche E-Mail-Adressen in der Community API Datei hinterlegen
+- Mehrere Kontakt-Kanäle in der Community API Datei hinterlegen, um nicht von einem Kanal abhängig zu sein (Mailserver down, Postfach voll)
+- Wissen und Zugriffsrechte im Team entsprechend verteilen
+- Urlaubsvertretung und Zuständigkeiten klar absprechen 
```

Comment:
puuh wir sind hier doch nicht auf Arbeit. Ich möchte niemandem zumuten, auch noch für Aktivitäten in der Freizeit Urlaubsvertretungsregelungen zu finden